source/scripts/model/Monster.js

```diff
@@ -17,7 +16,18 @@ export default class Monster {
 
         this.position = monster.position
         this.transition = true
-        this.movement = monster.protomonster.movement
+        this.movement = monster.protomonster.movement || function (target, current) {
+            var dx = target.x - current.x
+            var dy = target.y - current.y
+
+            if(Math.abs(dx) > Math.abs(dy)) {
+                if(dx > 0) return {x: +1}
+                if(dx < 0) return {x: -1}
+            } else {
+                if(dy > 0) return {y: +1}
+                if(dy < 0) return {y: -1}
+            }
+        }
```

Comment:
Ha! I like that this is now the default movement function! Especially since 80% of your monsters will be using it.

Out of curiosity, where is tpayne's pursuit functions? When will he be merging those in?